src/containers/Feed/feeds.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+import { Select } from 'antd';
+import { Link } from 'react-router-dom';
+
+import NavBar from '../../components/navigationBar';
+import JournalCard from '../../components/JournalCard';
+import { months, fakeData } from './data';
+import LogoHeader from '../../components/LogoHeader';
+import './feeds.css';
+
+const { Option } = Select;
+class Feed extends Component {
+  state = {
+    data: [],
+    monthCount: months,
+  };
+
+  componentDidMount() {
+    // fetch all data from firebase firstore
+    // async
+    // const result = await axios.get(`https:-------`);
+    // const cards = result.data.data[0];
+    // cards === fackData
+    //
+    const monthArray = fakeData.map(journal =>
+      moment(journal.timestamp).format('MMMM')
+    );
+    const filteredObject = monthArray.reduce((acc, curr) => {
+      if (typeof acc[curr] == 'undefined') {
+        acc[curr] = 1;
+      } else {
+        acc[curr] += 1;
+      }
```

Comment:
@AlaaTaima If you want, I think you could shorten this part (lines 30-34) to this:
`acc[curr] = (acc[curr] || 0) + 1`

Reply:
.. But it's not super important, just shorter :)